app/resonant-reference-app/models/Toolchain.js

```diff
@@ -132,16 +132,33 @@ let Toolchain = girder.models.ItemModel.extend({
     let self = this;
     let meta = self.get('meta');
     let options = {};
-    
+
+    // Figure out which options allow multiple fields
+    meta.mappings.forEach((mapping) => {
```

Comment:
If you like you can omit the parens here (in case you didn't know).

Unless there is a prevailing opinion that such parens should always be kept (similar to the C/C++ style rule about always keeping curly brackets in loops/conditionals).